verify_registration.py

```diff
@@ -0,0 +1,52 @@
+'''
+Write a Selenium script for your website that registers a new user, 
+in order to verify that registration works
+'''
+
+from selenium import webdriver
+from selenium.webdriver.common.by import By
+import random
+import string
+
+# Keeps webdriver from closes Chrome immediately
+options = webdriver.ChromeOptions()
+options.add_experimental_option('detach', True)
```

Comment:
Technically you do not want the browser to stay open. You only care about the browser to saying open when you are developing. Once you are done developing, the code will run in Jenkins or in some automated manner.
So you do not want the browser to stay open. 
In fact at the end of the script you should have 'driver.quite()' to close it.